packages/create-svelte/bin.js

```diff
@@ -75,8 +76,14 @@ async function main() {
 					message: 'Add type checking?',
```

Comment:
```suggestion
					message: 'Use TypeScript?',
```

Reply:
I personally liked the question being asking if someone wants type checking, because the two answers are different ways to fulfill that. To me, it's a little weird to say that you want TypeScript using TypeScript syntax